app/components/high-charts.js

```diff
@@ -79,14 +79,14 @@ export default Component.extend({
   callback: undefined,
 
   buildOptions: computed('chartOptions', 'content.[]', function() {
-    let theme = getWithDefault(this, 'theme', {});
-    let passedChartOptions = getWithDefault(this, 'chartOptions', {});
+    let theme = this.theme || {};
+    let passedChartOptions = this.chartOptions || {};
 
     let chartOptions = merge(theme, passedChartOptions);
-    let chartContent = get(this, 'content');
+    let chartContent = this.content;
 
     // if 'no-data-to-display' module has been imported, keep empty series and leave it to highcharts to show no data label.
-    if (!get(this, 'content.length') && !this.highcharts.Chart.prototype.showNoData) {
+    if (!this.content.length && !this.highcharts.Chart.prototype.showNoData) {
```

Comment:
again this breaks the semantics of what it was doing before: 

This will fix it: 
```suggestion
    if (!this.content?.length && !this.highcharts.Chart.prototype.showNoData) {
```